src/modeler/AutoRest.Swagger/Validation/CollectionObjectPropertiesNamingValidation.cs

```diff
@@ -34,7 +35,7 @@ public override IEnumerable<ValidationMessage> GetValidationMessages(Dictionary<
             foreach (var opPair in listOperations)
             {
                 // if the operation id is not of type _list* or does not return an array type, skip
-                if (!ListRegex.IsMatch(opPair.Value.OperationId) || !ValidationUtilities.IsXmsPageableOrArrayResponseOperation(opPair.Value, serviceDefinition))
+                if (!ListRegex.IsMatch(opPair.Value.OperationId) || !(opPair.Value.Extensions.GetValueOrNull(XmsPageable) != null))
```

Comment:
since we already have a method defined in utilities around this, but the issue is that it does 2 things at the same time, could we split it, and call the one that we need here, this way "x-ms-pageable" string doesn't need to be defined here, and we try to centralize concept definitions.?